integrationExamples/gpt/haloRtdProvider_example.html

```diff
@@ -232,17 +245,33 @@
                     auctionDelay: 1000
                 },
                 realTimeData: {
-                    auctionDelay: 1000,
-                    dataProviders: [{name: "audigent", waitForIt: true}]
+                    auctionDelay: testAuctionDelay, // lower in real scenario to meet publisher spec
+                    dataProviders: [
+                        {
+                            name: "halo",
+                            waitForIt: true,
+                            params: {
+                                mapSegments: {
+                                    appnexus: true
+                                },
+                                segmentCache: false,
+                                requestParams: {
+                                    publisherId: 0
+                                }
+                            }
+
+                        }
+                    ]
                 }
             });
             pbjs.addAdUnits(adUnits);
+
             pbjs.requestBids({bidsBackHandler: sendAdserverRequest});
         });
 
         function sendAdserverRequest() {
             document.getElementById('tdid').innerHTML = adUnits[0].bids[0].userId['tdid'];
```

Comment:
![image](https://user-images.githubusercontent.com/12955750/98942241-059e2a00-24ee-11eb-8b76-1e5737d0961d.png)

Reply:
added a test id that does the full retrieval cycle for appnexus

Reply:
am I doing something wrong here, I'm running this test page and still get exceptions:
![image](https://user-images.githubusercontent.com/12955750/99096019-1c1fb080-25d6-11eb-8850-ab3059b9f60d.png)

Reply:
can you try refreshing the page and tell me if you receive the error?  There might have been a timeout taking place.